public/app/features/dashboard/dashnav/dashnav.html

```diff
@@ -3,7 +3,7 @@
 	<div>
 		<a class="navbar-page-btn" ng-click="ctrl.showSearch()">
 			<i class="gicon gicon-dashboard"></i>
-			{{ctrl.dashboard.title}}
+			<span ng-if="ctrl.dashboard.meta.folderTitle !== 'General'" class="navbar-page-btn--folder">{{ctrl.dashboard.meta.folderTitle}} / </span>{{ctrl.dashboard.title}}
```

Comment:
I think it's better to check folderId > 0 than to compare folderTitle

Reply:
Sounds like an idea. Does the folderId incerement? Maybe just check folderId != 0

Reply:
yes, it increments. folderId > 0 is the most idiot proof. And easier to reason about a positive condition than a negative.